docs/marketplace/creators-hub/review-dispute-guidelines.md

```diff
@@ -0,0 +1,47 @@
+---
+slug: review-dispute-guidelines
+title: Review Dispute Guidelines
+description: Learn about FlutterFlow Marketplace review dispute process and when reviews may be removed or modified.
+tags: [MarketPlace, Review Dispute, Guidelines]
+sidebar_position: 3
+keywords: [FlutterFlow, MarketPlace, Review Dispute, Guidelines, Feedback, Reviews]
+---
+
+# FlutterFlow Marketplace Review Dispute Guidelines
+
+At FlutterFlow Marketplace, we believe in transparent and honest feedback. Reviews are an essential part of helping buyers make informed decisions and helping creators improve their work.
+
+However, not all reviews are created equal. Sometimes feedback is based on misunderstandings, irrelevant factors, or issues unrelated to the quality of the item itself. This guideline outlines when and how we handle review disputes.
+
+### When We *Will* Remove a Review
+
+We may remove a review if it meets **one or more** of the following criteria:
+
+- **Spam or Abuse:** The review contains offensive language, harassment, or unrelated spam content.
+- **Misuse of the Review System:** The review is about unrelated topics (e.g., FlutterFlow features, pricing, unrelated bugs).
+- **Critical Misunderstanding:** The review is based on a clear misunderstanding of the item's purpose or scope, despite the listing being accurate and transparent.
+- **Irrelevant to the Current Version:** The review references issues that have since been resolved, and the creator has updated the item significantly.  
+    *Note: We may remove outdated reviews in cases where leaving them would misrepresent the current product.*
```

Comment:
Notes, tips are usually in admonitions 
So like - 

:::note[heading]
the full text
:::

remove [heading] completely if there is no heading.